### DISCUSSION TITLE
Can't send email (using TLS / Port 587 / Gmail SMTP)

### DISCUSSION THREAD

**Discussion by fcecin** [requarks/wiki#2801] [Help / Questions]
**Question**

I'm getting the "139799482002760:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:" error when trying to use the Gmail SMTP server to send a test email from the Mail configuration screen.

The credentials I'm using seem OK, and I can send email using them and the 'sendemail' command-line tool from my own local machine:

sendemail -f fabianasendmail@gmail.com -t fabianasendmail@gmail.com -u testtest -m "test message" -s smtp.gmail.com:587 -o tls=yes -xu fabianasendmail -xp 'MYREDACTEDPASSWORD'

This is with a fresh DigitalOcean Marketplace Wiki.js docker image deployment. I auto-updated the wiki from a 2.4 version to the latest 2.5 by clicking on the update button in the admin interface, as the Docker image comes with the 2.4 version of the wiki, but that was about it.

I've attached a screenshot of my configuration screen. I click to "Apply" settings every time I change anything. 
![Screenshot from 2020-12-08 18-50-55](https://user-images.githubusercontent.com/1266227/101546406-7e769180-3987-11eb-85b7-f9962e115195.png)
 
Toggling the "Verify SSL certificate" option doesn't fix it.

Using "fabianasendmail@gmail.com" as the SMTP username instead of "fabianasendmail" also doesn't change anything.

There is definitely some network traffic going on between the wiki machine and the SMTP server every time I click the button to send out the test email, as per iftop.

I've seen similar issues closed due to mail configuration error, but I'm out of ideas as to how to present my e-mail account credentials in the admin interface.

I tried every variation of the credentials and clicking "apply" and trying again that I can think of, but no change.

Any ideas?

**Host Info:**
OS: 18.04.4 LTS, Docker (using DigitalOcean Marketplace image)
Wiki.js version: 2.5.170
Database engine: Postgres 11.8

**Comment by mihailim** [comment]
Does it work with the "Secure (TLS)" option turned _off_? The description under it clearly states it should be turned off when using port 587 (the client should then negotiate using STARTTLS.)
